<launch>
  <!-- Argument for device id -->
  <arg
    name="cam1"
    value="10"
  />
  <!-- Publisher node -->
  <node pkg="cv_camera" type="cv_camera_node" name="left">
  	<param name="frame_id" value="virtual_camera_left" />
  	<param name="device_id" value="10" />
  	<remap from="left/image_raw" to="stereo/left/image_raw" />
  </node>
 
  <!-- Subscriber node -->
<!--
<node
    pkg="image_proc"
    type="image_proc"
    name="image_proc" />
-->
</launch>
